src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/OcrValidator.java

```diff
@@ -47,8 +49,14 @@ public OcrValidator(
     }
     //endregion
 
-    public void assertIsValid(InputEnvelope envelope) {
-        Optionals.ifAllPresent(
+    /**
+     * If required, validates the OCR data of the given envelope.
+     *
+     * @return List of warnings for valid OCR data, to be displayed to the caseworker
+     * @throws OcrValidationException if the OCR data is invalid
+     */
+    public List<String> assertIsValid(InputEnvelope envelope) {
```

Comment:
How about firstly making changes in db model (or servicebus message) and on second PR - enabling this method to store **and** send warnings?